example/spec/parallel/ingestGranule/IngestGranuleDuplicateHandlingSpec.js

```diff
@@ -134,7 +136,18 @@ describe('When the Ingest Granules workflow is configured\n', () => {
 
       it('MoveGranules outputs', async () => {
         lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'MoveGranules');
-        currentFiles = await getFilesMetadata(lambdaOutput.payload.granules[0].files);
+        // Logging to catch intermittent failures
+        const outputGranule = lambdaOutput.payload.granules[0];
+        currentFiles = await Promise.all(outputGranule.files.map((file) => {
+          log.info(`Getting metadata of ${file.bucket}/${file.filename} in s3`);
```

Comment:
As discussed with @ifestus, we could also put this logging in `getFileMetadata` itself and then the code in the test wouldn't have to change. Nor do I think we'd need to worry about following up to remove the logging later, because it is generally useful. However, it's not blocking